app/jobs/geocode_venues_job.rb

```diff
@@ -0,0 +1,17 @@
+class GeocodeVenuesJob < ActiveJob::Base
+  queue_as :default
+
+  def perform
```

Comment:
Needs error handling. If a venue fails to get geolocated, log and `next`, please.